pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1604,6 +1608,13 @@ type ObjectEndpoints struct {
 	Secure []string `json:"secure"`
 }
 
+// ObjectStoreHostingSpec represents the hosting settings for the object store
+type ObjectStoreHostingSpec struct {
+	// The virtual host path where the bucket can be hosted
```

Comment:
Add more info to this so that users using `kubectl explain` (or CRD helper programs) can get fuller info about what to do. I would just copy-paste the content from the object store CRD doc for ease.